src/EditorFeatures/CSharpTest/EncapsulateField/EncapsulateFieldTestState.cs

```diff
@@ -27,11 +27,12 @@ internal class EncapsulateFieldTestState : IDisposable
         public Document TargetDocument { get; }
         public string NotificationMessage { get; private set; }
 
-        private static readonly ComposableCatalog s_composableCatalog =
-            TestExportProvider.MinimumCatalogWithCSharpAndVisualBasic.WithParts(
-            typeof(CSharpEncapsulateFieldService),
-            typeof(EditorNotificationServiceFactory),
-            typeof(DefaultDocumentSupportsFeatureService));
+        private static readonly IExportProviderFactory s_exportProviderFactory =
+            ExportProviderCache.CreateExportProviderFactory(
+                TestExportProvider.MinimumCatalogWithCSharpAndVisualBasic.WithParts(
```

Comment:
Naming here is again throwing me for a loop: it's odd that the creation of a _factory_ uses things from a _provider_. Should this really be TestCatalogs.MinmumCatalogWithCSharpAndVisualBasic?

I get the naming isn't new here, but since we're trying to make this correct, this would help ensure the concepts are clear for the future. Is that a big change?

Reply:
➡️ We have a few inconsistencies in naming, but not enough to be a major concern for me at this time. We can file a follow-up issue if you want.